src/test/emwasm_compile_known_gcc_test_failures.txt

```diff
@@ -51,13 +45,10 @@
 
 
 # Computed gotos (but works with bare wasm clang)
-	20041214-1.c O0
-	20071220-1.c
+	20071220-1.c O3
 	20071220-2.c
```

Comment:
Why do we still have computed gotos?

Reply:
Turns out if you do a blockaddress without an indirectbr the indirectbr lowering pass doesn't lower it away, which makes sense. That still can't really be supported in wasm (e.g. there is no address per se, so no use of the block address other than as a branch target makes sense).